cli/src/com/redhat/ceylon/common/tools/AbstractTestTool.java

```diff
@@ -92,9 +92,11 @@ public void setVersion(String version) {
         this.version = version;
     }
     
-    @Option(longName = "tap")
-    @Description("Enables the Test Anything Protocol v13.")
-    public void setTap(boolean tap) {
+    @OptionArgument(argumentName = "file")
```

Comment:
> which, in my opinion, is how most tests should behave

I don't think we can count, that nobody will do it.

> But then I think it’s inconsistent that the HTML option is called just --report instead of --html.

yeah, that's pity, not sure if it's worthwhile to add `html` option and mark `report` option as deprecated (or remove completely)

> What default file name would you suggest? reports/tap?

here is how it does html reporter https://github.com/ceylon/ceylon-sdk/blob/master/source/ceylon/test/reporter/HtmlReporter.ceylon#L40-L45 (the subdir parameter is either `test` or `test-js`